src/main/resources/javascripts/src/mount.js

```diff
@@ -0,0 +1,13 @@
+const jsdom = require('jsdom');
+
+const { JSDOM } = jsdom;
+
+function mount(content) {
+  const dom = new JSDOM(content);
+  const { window } = dom;
+  const { document } = window;
+
+  return document.documentElement;
+}
+
+module.exports = mount;
```

Comment:
possible small (untested) refactoring

```suggestion
const { JSDOM } = require('jsdom');

module.exports = content => new JSDOM(content).window.document.documentElement;
```

Reply:
Thanks @oscarduignan. If I'm honest, I prefer the version with the named function and the de-structuring - it's just a little readable for me! 😅

Reply:
(sorry didn't see the notification for this yesterday)